tests/plugins/test_drdk.py

```diff
@@ -6,9 +6,9 @@
 class TestPluginDRDK(unittest.TestCase):
     def test_can_handle_url(self):
         should_match = [
-            'https://www.dr.dk/tv/live/dr1',
```

Comment:
put the old urls into `test_can_handle_url_negative`

also remove the **yes** for vods in `docs/plugin_matrix.rst`